testing_and_setup/compass/ocean/jigsaw_to_MPAS/mesh_definition_tools.py

```diff
@@ -11,13 +11,41 @@
 
 import numpy as np
 import matplotlib.pyplot as plt
+import xml.etree.ElementTree as ET
+from matplotlib.colors import LinearSegmentedColormap
 plt.switch_backend('agg')
 
 
 ##########################################################################
 # Functions
 ##########################################################################
 
+def read_xml_colormap(xmlFile, mapName):
+    """Read in an XML colormap"""
```

Comment:
I copied this function from `build_mesh.py`. I think this `mesh_definitions_tools.py` file is for lower-level subroutines. Should we remove the original, and move it here instead, or just leave it both places? Why did yours start with an underscore?

Reply:
That's fine.  Yes, we should definitely remove the redundant copy.  You will need the other function for registering the color map if you didn't copy it.

The underscore is a convention to indicate that it's a private function that users aren't intended to use directly and which might change its API without warning.  In this case, I just copied that form MPAS-Analysis, where it is a helper function for a public function that registers all the color maps that MPAS-Analysis supports.